src/corona_hakab_model/consts.py

```diff
@@ -105,9 +119,21 @@ class Consts(NamedTuple):
     detection_pool: List[DetectionSettings] = [
         DetectionSettings(
             name="hospital",
-            detection_test=DetectionTest(detection_prob=0.98,
-                                         false_alarm_prob=0.,
-                                         time_until_result=3),
+            detection_test=DetectionTest({
```

Comment:
add those variables to simulation_parameters_example.py

Reply:
done